src/java/test/org/apache/zookeeper/server/quorum/ReconfigRecoveryTest.java

```diff
@@ -28,9 +28,15 @@
 import org.apache.zookeeper.test.ClientBase;
 import org.apache.zookeeper.test.ReconfigTest;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
 public class ReconfigRecoveryTest extends QuorumPeerTestBase {
+    @Before
+    public void setup() {
+        QuorumPeerConfig.setReconfigEnabled(true);
```

Comment:
Yes if those tests require reconfig feature being enabled. I think I've patched all reconfig test in ZOOKEEPER-2014, and the modifications of related tests here are all about refactoring and consolidating setting of the flag in a single place.